editions/tw5.com/tiddlers/widgets/EditTextWidget.tid

```diff
@@ -30,7 +30,8 @@ The content of the `<$edit-text>` widget is ignored.
 |size |The size of the input field (in characters) |
 |autoHeight |Either "yes" or "no" to specify whether to automatically resize `textarea` editors to fit their content (defaults to "yes") |
 |minHeight |Minimum height for automatically resized `textarea` editors, specified in CSS length units such as "px", "em" or "%" |
-|rows|Sets the rows attribute of a generated textarea |
+|rows |Sets the rows attribute of a generated textarea |
+|storeTitle <<.from-version 5.1.22>> |An additional tiddler title that stores the user's input |
```

Comment:
The `<<.from-version 5.1.22>>` should go immediately after the second vertical bar for consistency with usage elsewhere. I also try to avoid apostrophes in documentation like this because I've no confidence that I get them right. Perhaps here better to say "An optional tiddler title that is updated with the text content each time it is directly updated by the user". Ideally, we'd also say "This can be used to create keyboard driven autocomplete inputs, please see XXXX", with a new tiddler giving instructions to end users wanting to build their own implementations.